contracts/tokenomics/generator/src/contract.rs

```diff
@@ -1353,6 +1358,16 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> Result<Binary, ContractErro
             lp_token,
             future_block,
         )?)?),
+        QueryMsg::ListOfStakers {
+            lp_token,
+            start_after,
+            limit,
+        } => Ok(to_binary(&query_list_of_stakers(
```

Comment:
Ok() is redundant

Reply:
We need to use Ok, cuz to_binary return StdErr but we are expect ContractError.

Reply:
Change return type to StdResult<Binary> same as we do in other contracts